index.js

```diff
@@ -0,0 +1,22 @@
+require('dotenv').config()
+const express = require('express')
+const authRouter = require('./routes/authRouter')
+const mongoose = require('mongoose')
+
+const PORT = process.env.PORT || 5000;
+const app = express()
+
+app.use(express.json())
+app.use('/auth', authRouter)
+const start = async () => {
+	try {
+		await mongoose.connect(`mongodb+srv://Jamsclub:jamsclub02@jamscluster.rxh4hgq.mongodb.net/?retryWrites=true&w=majority`)
```

Comment:
This should be in env variables and not as a template string
```suggestion
		await mongoose.connect(process.env.DB_CONNECTION_STRING)
```